robot/EDA/resources/AffiliationsSettingsPageObject.py

```diff
@@ -1,41 +1,30 @@
-from cumulusci.robotframework.pageobjects import ListingPage
+from cumulusci.robotframework.pageobjects import BasePage
 from cumulusci.robotframework.pageobjects import pageobject
 from locators import affiliations_locators
+from locators import eda_lex_locators
 from selenium.webdriver.common.keys import Keys
 
 
-@pageobject("Listing", "hed__HEDA_Settings")
-class EDASettingsPage(ListingPage):
-    object_name = "hed__HEDA_Settings"
+@pageobject("Affiliations", "HEDA_Settings")
+class AffiliationsSettingsPage(BasePage):
 
     @property
     def eda(self):
         return self.builtin.get_library_instance('EDA')
 
     def _is_current_page(self):
-        """ Verify we are on the EDA Settings page
-            by verifying that the header title is 'EDA Settings'
         """
-        locator = affiliations_locators["header"]
-        self.selenium.page_should_contain_element(
-            locator,
-            message="Header with text 'EDA Settings' is not available on the page"
-        )
+            Verify we are on the EDA Settings page for Affiliations
+            by verifying the HEDA Settings URL and the Affiliations tab
+        """
+        location = "/lightning/n/{}{}".format(self.eda.get_eda_namespace_prefix(), self._object_name)
+        self.selenium.location_should_contain(location)
 
-    def _go_to_page(self, filter_name=None):
-        url_pattern = "{root}/lightning/n/{object}"
-        name = self._object_name
-        object_name = "{}{}".format(self.cumulusci.get_namespace_prefix(), name)
-        url = url_pattern.format(root=self.cumulusci.org.lightning_base_url, object=object_name)
-        self.selenium.go_to(url)
-        self.salesforce.wait_until_loading_is_complete()
-        self.eda.wait_for_locator("frame", "accessibility title")
-        self.salesforce.select_frame_with_title("accessibility title")
```

Comment:
is removing go to page keyword intentional?

Reply:
Yes. There are about 6-7 settings pages and the way to navigate to them is same. So I created a keyword in EDA.py as go_to_eda_settings_tab. This will take the arg as "Affiliations" to go to the AffiliationsSettings page.